### ISSUE TITLE
Blueprint Language Server - Basic Implementation

### ISSUE THREAD

**Issue by fr3shw3b** [two-hundred/celerity#41]
Add an implementation of the Blueprint Language Server that has some basic diagnostics functionality that validates a blueprint spec on document change and load events and returns/pushes diagnostics derived from blueprint spec validation results.

**Comment by fr3shw3b** [comment]
WIP Diagnostics integration so far, this will be finalised (as a part of a separate issue) once all the core validation in the blueprint library is complete
